docs/buildah-config.md

```diff
@@ -53,7 +53,7 @@ using the specified container.
 
 Note: this setting is not present in the OCIv1 image format, so it is discarded when writing images using OCIv1 formats.
 
-**--entrypoint** *entry*
+**--entrypoint** *"string"* | *'["array_entry1", "array_entry2", ...]'*
```

Comment:
How about 
**--entrypoint** *"string"* | *'["command", "arg1", ...]'*

Reply:
Cheers. Done.

Reply:
For consistency, should "string" be "command" also?

Reply:
Yes I was thinking that also.

Reply:
I think only 

**--entrypoint** "command" "arg1",...

is the best.

Reply:
@ripcurld this doesn't show that entrypoint takes string _or_ array.

Reply:
@ripcurld I agree with @pixdrift 
we are trying to show that --entrypoint takes input as either a simple command string, or as an array of command followed by argumets '[ "command", "arg1", "arg3" ]' 

--entrypoint "command" "arg1" 
is invalid


--entrypoint "command"
or 
--entrypoint  '[ "command", "arg1", "arg3" ]' 
Are valid